tools/logCollection/output/sample_tdr_logs_2025-05-01 10:51:09.527812.csv

```diff
@@ -0,0 +1,2 @@
+_time,src_ip,dest_ip,dest_port,user_name,user_id,user_id_provider,session_id,method,url,app,http_user_agent,status,http_content_type,bytes,duration,nih_ico,cadr_name,user_country_name,user_org,user_email,associated_study,eRA_commons_id,user_permission_group,event_type,dataset_id,snapshot_id

```

Comment:
Do you need this output? Should we mark output as ignored?

Reply:
Indeed, I think this should be excluded!

Reply:
Oh, okay. I thought a sample could be helpful to understanding the desired output if someone runs into issues with the script, but it would be nice to include this whole folder in gitignore. I can do that.